### PR TITLE
enhancement/86eqdy51f-Contact-Us-API-Integration

### PR DESCRIPTION
**Pull request by nasolgabriel**
**Contact US Creation API integration**

- Integrated the Contact Us form with the  API.
- Displayed success and error toasts based on the API response.

**Screenshot:**
image

### CODE REVIEW

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx [hunk 1]
```diff
@@ -3,7 +3,10 @@ import { ContactForm } from "./ContactForm";
 import { yupResolver } from "@hookform/resolvers/yup";
```
Comment (BryanPalad) [inline apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx:3]: Kindly add this property comment above all your modified files..

/**
* Property of the NCLEX Power.
* Reuse as a whole or in part is prohibited without permission.
* Created by the Software Strategy & Development Division
*/

**Review by BryanPalad — CHANGES_REQUESTED** [review]

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx [hunk 2]
```diff
@@ -13,19 +16,30 @@ export function ContactFormBlock() {
...
-      false
-    );
-
-    reset();
-  };
+  async function onSubmit(params: ContactFormType) {
+    try {
+      await mutateAsync({ ...params });
+      console.log(params);
+      toast.executeToast(
```
Comment (BryanPalad) [inline apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx:28]: You can add the type of toast here like success or type: "error" if failed.
toast.executeToast(
        "Your message has been received. Thank you",
        "top-right",
        false, {type: "success"}
      );

**Review by BryanPalad — CHANGES_REQUESTED** [review]

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx (hunk 1 above)
Comment (nasolgabriel) [reply apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx:3]: will do it immediately, thank you

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx [hunk 3]
```diff
@@ -13,19 +16,30 @@ export function ContactFormBlock() {
...
-      "top-right",
-      false
-    );
-
-    reset();
-  };
+  async function onSubmit(params: ContactFormType) {
+    try {
+      await mutateAsync({ ...params });
+      console.log(params);
```
[the remark this answers is not in the document]
Comment (nasolgabriel) [reply apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx:27]: working on it. TY

**Pull request closed without merging**